docs/csharp/programming-guide/classes-and-structs/toc.yml

```diff
@@ -7,41 +7,42 @@
     href: objects.md
```

Comment:
I tested these on the review site.  There's an interesting side effect when we have sub-TOCs. 

We actually *want* the default behavior of expanding the TOC. Otherwise, on the child TOC, there's no root node.